src/algorithms/controllers/PredictiveStanleyController.cpp

```diff
@@ -377,35 +315,22 @@ namespace controllers
     }
 
     /******************************************************************************
-     * @brief Accessor for the steering angle limit of the stanley controller.
-     *
-     * @return double - The steering angle limit for the controller.
-     *
-     * @author clayjay3 (claytonraycowen@gmail.com)
-     * @date 2025-01-11
-     ******************************************************************************/
-    double PredictiveStanleyController::GetSteeringAngleLimit() const
-    {
-        return m_dSteeringAngleLimit;
-    }
-
-    /******************************************************************************
-     * @brief Accessor for the wheelbase of the stanley controller.
+     * @brief Accessor for the angular velocity limit of the stanley controller.
      *
-     * @return double - The wheelbase of the controller.
+     * @return double - The angular velocity limit for the controller.
      *
      * @author clayjay3 (claytonraycowen@gmail.com)
      * @date 2025-01-11
      ******************************************************************************/
-    double PredictiveStanleyController::GetWheelbase() const
+    double PredictiveStanleyController::GetAngularVelocityLimit() const
     {
-        return m_dWheelbase;
+        return m_dAngularVelocityLimit;
     }
 
     /******************************************************************************
      * @brief Accessor for the reference path that the controller is following.
      *
-     * @return std::vector<geoops::Waypoint> - A copy of the reference path.
+     * @return std::vector<geoops::Waypoint> - A codPY of the reference path.
```

Comment:
Fix the capitalization of the return comment block

```suggestion
     * @return std::vector<geoops::Waypoint> - A copy of the reference path.
```